### DISCUSSION TITLE
Field editor options for "Save & Add Another" and "Save as New Field"

### DISCUSSION THREAD

**Discussion by Mosnar** [craftcms/cms#9614] [Ideas] [answered]
The option to quickly "Save & Add Another" and "Save as New Field" would save so much time when first setting up a bunch of fields in the CMS. I'm almost never adding a single field, almost always several fields at once. These are options that already exist for elements, so it should feel pretty natural to exist in other places as well.

Bonus points for remembering what field group I had selected prior to saving and adding another (similar to how entries remember their parents).

**Comment by brandonkelly** [comment, marked as answer]
Edit Field pages will have a “Save and add another” action starting in Craft 4.6 (f11b1ecf63c2411b53fecabe3bd6c3f2b4f54cb2).

“Save as a new field” would be a little tricky, as some field types like Matrix store their own settings (block types) beyond the JSON blob stored in the `fields` table. So it would need to be an opt-in thing. And I’m guessing it won’t be as needed in Craft 5 thanks to field instances.
